src/pages/api/sendEmail.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable prettier/prettier */
```

Comment:
Why disable prettier? It's generally better to allow Prettier to format the code for consistency and readability.

Reply:
whoopsies copied that in from the index page, it was prettier disabled